index.html

```diff
@@ -120,18 +119,20 @@ <h2 class="h2-work text-color-1 no-margin-bottom">
             <div class="flex-center">
               <h2 class="work-title incard-text-color">Multi-Post Stories Gain+Glory</h2>
             </div>
-            <!--skils-->
+
             <div>
               <ul class="skils">
-                <li class="skils-border incardskills-button-color skils-txt incard-text-color one-skil">Ruby on rails</li>
+                <li class="skils-border incardskills-button-color skils-txt incard-text-color one-skil">Ruby on rails
+                </li>
                 <li class="skils-border incardskills-button-color skils-txt incard-text-color two-skil">Css</li>
-                <li class="skils-border incardskills-button-color skils-txt incard-text-color three-skil">Javascript</li>
+                <li class="skils-border incardskills-button-color skils-txt incard-text-color three-skil">Javascript
+                </li>
                 <li class="skils-border incardskills-button-color skils-txt incard-text-color four-skil">html</li>
               </ul>
             </div>
-            <!--skils-->
+
             <button class="btn-success button-color inbutton-text-color">
-              <span class="text-inside-button "> See Project</span></button>
+              <span class="text-inside-button ">See Project</span></button>
```

Comment:
- [ ] As this button will be used to give more details on your projects, kindly, let's use a link `a` or `button` tag, 🙏

Reply:
I have already used a button tag in this line of code, If you please explain this required change then it would be very helpful. thanks